library/src/G4CMPCrystalGroup.cc

```diff
@@ -141,15 +140,48 @@ G4bool G4CMPCrystalGroup::FillOrthorhombic(G4double Cij[6][6]) const {
 
 G4bool G4CMPCrystalGroup::FillRhombohedral(G4double Cij[6][6]) const {
   G4double C11=Cij[0][0], C12=Cij[0][1], C13=Cij[0][2], C14=Cij[0][3];
-  G4double C15=Cij[0][4], C33=Cij[2][2], C44=Cij[3][3], C66=0.5*(C11-C12);
-
-  Cij[1][1] = C11;	// Copy small number of individual elements
-  Cij[1][2] = C13;
-  Cij[1][3] = -C14;
-  Cij[1][4] = -C15;
-  Cij[3][5] = -C15;
-  Cij[4][4] = C44;
-  Cij[4][5] = C14;
```

Comment:
This looks like a substantial bug fix?  Thank you!  In your dated comment at the top, could you please provide a _narrative_ explanation of this, rather than just naming the function.